beancount_dkb/ec.py

```diff
@@ -224,9 +224,9 @@ def _update_meta(self, meta: Dict[str, str]):
                 # assertions work.
 
                 self._balance_amount = Amount(
-                    fmt_number_de(value.value.rstrip(" EUR")), self.currency
+                    fmt_number_de(value.value.split()[0]), self.currency
                 )
                 self._balance_date = datetime.strptime(
-                    key.lstrip("Kontostand vom ").rstrip(":"), "%d.%m.%Y"
+                    key.removeprefix("Kontostand vom ").rstrip(":"), "%d.%m.%Y"
```

Comment:
Looks like `removeprefix` was added in 3.9 which is why the build for 3.8 fails.